lnwire/lnwire.go

```diff
@@ -194,6 +208,18 @@ func WriteElement(w *bytes.Buffer, element interface{}) error {
 		if _, err := w.Write(e[:]); err != nil {
 			return err
 		}
+
+	case WarningData:
```

Comment:
is this called anywhere?

Reply:
I think we can actually also just fall through to the case below.

Reply:
No, and the method will be removed soon* (YY comment). However, I added in case someone is using `lnwire.WriteElement` outside the lnd code base for completeness.